src/PlatformAPI.ts

```diff
@@ -185,7 +185,7 @@ export interface PlatformAPI {
   sendReadReceipt: (threadID: ThreadID, messageID?: MessageID, messageCursor?: string) => Awaitable<void>
 
   addReaction?: (threadID: ThreadID, messageID: MessageID, reactionKey: string) => Awaitable<void>
-  removeReaction?: (threadID: ThreadID, messageID: MessageID, reactionKey: string) => Awaitable<void>
+  removeReaction?: (threadID: ThreadID, parentMessageID: MessageID, reactionKey: string, id: string, messageID?: string) => Awaitable<void>
```

Comment:
```suggestion
  removeReaction?: (threadID: ThreadID, parentMessageID: MessageID, reactionKey: string, reactionID?: string, messageID?: string) => Awaitable<void>
```

more consistent naming